app/components/Views/confirmations/components/modals/switch-account-type-modal/account-network-row/account-network-row.tsx

```diff
@@ -32,10 +42,15 @@ const AccountNetworkRow = ({
   address: Hex;
   network: EIP7702NetworkConfiguration;
 }) => {
+  const useMultichainAccountDesign = useSelector(
```

Comment:
Nit: But maybe use `useMultichainAccountsDesign` instead?

```suggestion
  const useMultichainAccountsDesign = useSelector(
```